src/platform/intel/cavs/include/cavs/drivers/dw-dma.h

```diff
@@ -51,12 +52,27 @@
 #define DW_CFG_LOW_DEF		0x3
 #define DW_CFG_HIGH_DEF		0x0
 
+/* LLPC address */
+#define DW_CHLLPC(dma, chan) \
+	SHIM_GPDMA_CHLLPC((dma)->plat_data.id, (chan)->index)
+
 #define platform_dw_dma_set_class(chan, lli, class) \
 	(lli->ctrl_hi |= DW_CTLH_CLASS(class))
 
 #define platform_dw_dma_set_transfer_size(chan, lli, size) \
 	(lli->ctrl_hi |= (size & DW_CTLH_BLOCK_TS_MASK))
 
+#define platform_dw_dma_llp_config(dma, chan, config) \
```

Comment:
Any reason why these are not static inline ? We get the type checking and can pass if pointers if needed later.

Reply:
@lgirdwood Changed.